shared/actions/config/index.tsx

```diff
@@ -292,30 +294,32 @@ const showDeletedSelfRootPage = () => [
   RouteTreeGen.createNavigateAppend({path: [Tabs.loginTab]}),
 ]
 
-const switchRouteDef = (
```

Comment:
in the new nav we can dynamically have different routes when we render, so we don't need to explicitly switch stacks. this is how we can deprecate the 'Main' route